src/styles/_bind-service.less

```diff
@@ -0,0 +1,55 @@
+.bind-form {
+  form {
+    margin-bottom: 15px;
+  }
+
+  h3 {
+    margin-top: 0;
+  }
+
+  input[type="checkbox"] {
```

Comment:
i dont love that this will affect any checkboxes ever added to the bind form, can you do something more specific to this particular input.  I assume the issue is that its at the top of the form?